tap_egencia/schemas/get_transactions.py

```diff
@@ -0,0 +1,161 @@
+        Property("policy_reason_description", StringType),
+    )
+
+
+class priceObject(CustomObject):
+    properties = PropertiesList(
+        Property("average_leg_amount", StringType),
+        Property("average_leg_price", StringType),
+        Property("average_segment_amount", StringType),
+        Property("base", StringType),
+        Property("base_amount", StringType),
+        Property("best_fare_option", StringType),
+        Property("change_fee", StringType),
+        Property("change_fee_amount", StringType),
+        Property("change_fees", StringType),
+        Property("coupon_amount", StringType),
+        Property("extra_guest_charges", StringType),
+        Property("extra_person_amount", StringType),
+        Property("fare_bases", StringType),
+        Property("fee", StringType),
+        Property("fees", StringType),
+        Property("flight_amount", StringType),
+        Property("goodwill_amount", StringType),
+        Property("leg_amount", StringType),
+        Property("lowest_logical_fare", StringType),
+        Property("published_fare", StringType),
+        Property("segment_amount", StringType),
+        Property("tax", StringType),
+        Property("tax_gst", StringType),
+        Property("tax_hst", StringType),
+        Property("tax_qst", StringType),
+        Property("tax_vat", StringType),
+        Property("taxes", StringType),
+        Property("total", StringType),
+        Property("transaction_amount", StringType),
+        Property("transaction_amount_tax_gst", StringType),
+        Property("transaction_amount_tax_hst", StringType),
+        Property("transaction_amount_tax_qst", StringType),
+        Property("transaction_amount_vat", StringType),
+        Property("trip_amount", StringType),
+        Property("true_ticket_amount", StringType),
+    )
+
+
+class travelDatesObject(CustomObject):
+    properties = PropertiesList(
+        Property("travel_end_date", StringType),
+        Property("travel_start_date", StringType),
+    )
+
+
+class travelerObject(CustomObject):
+    properties = PropertiesList(
+        Property("email", StringType),
+        Property("group", StringType),
+        Property("is_guest", StringType),
+        Property("meeting_attendee_group", StringType),
+        Property("name", StringType),
+    )
+
+
+class transactionsObject(CustomObject):
+    properties = ArrayType(
+        PropertiesList(
+            Property("advance_purchase_days", StringType),
+            Property("advance_purchase_window", StringType),
+            Property("ancillary_type", StringType),
+            Property("booking_method", StringType),
+            Property("cabin_class", StringType),
+            Property("cabin_class_name", StringType),
+            Property("class_of_service", StringType),
+            Property("client_code", StringType),
+            Property("company_name", StringType),
+            # Property("custom_data_fields", ObjectType),
```

Comment:
Do we need to know the structure of this object in order to return it, or can we treat it as a generic object? If we don't need to know, we should go ahead and include this even if we don't know exactly what it will look like because we want to return the raw data as completely as possible.

Reply:
We need to define the structure of what we are pulling. omitting any fields that are returned throws an error. I had to re-add the custom_data_fields today because it was breaking.